supplemental_content/admin.py

```diff
@@ -47,6 +50,11 @@ class SectionAdmin(BaseAdmin):
     search_fields = ["title", "part", "section_id"]
     ordering = ("title", "part", "section_id", "parent")
 
+    def formfield_for_foreignkey(self, db_field, request, **kwargs):
```

Comment:
Does this belong in BaseAdmin?  I feel like in some cases this won't be the right thing, even if it is the right thing today.